pkg/k8s/api.go

```diff
@@ -133,7 +133,7 @@ func (kubeAPI *KubernetesAPI) GetReplicaSets(namespace string) ([]appsv1.Replica
 // GetPodStatus receives a pod and returns the pod status, based on `kubectl` logic.
 // This logic is imported and adapted from the github.com/kubernetes/kubernetes project:
 // https://github.com/kubernetes/kubernetes/blob/33a3e325f754d179b25558dee116fca1c67d353a/pkg/printers/internalversion/printers.go#L558-L640
-func GetPodStatus(pod *corev1.Pod) string {
+func GetPodStatus(pod corev1.Pod) string {
```

Comment:
For my own understanding, what was the reason for the change here?

Reply:
It's a really boring reason.  In `checkControllerRunning` we process a slice of Pods (not Pod pointers) and wish to call this function.  We could simply pass a reference to the pod into this function and it would work fine, but our linter doesn't like that we'd be taking the reference to a loop variable and complains about it.

Reply:
Cool, thanks for the explanation.